tests/unit/test_service.py

```diff
@@ -261,8 +261,8 @@ def _fake_measurement_function():
                 "ni.measurementlink.measurement.v2.MeasurementService",
             ],
             {
-                "ni/service.description": '"Measure inrush current with a shorted load and validate results against configured limits."',
-                "ni/service.collection": '"CurrentTests.Inrush"',
+                "ni/service.description": "Measure inrush current with a shorted load and validate results against configured limits.",
+                "ni/service.collection": "CurrentTests.Inrush",
                 "ni/service.tags": '["powerup","current"]',
```

Comment:
It would be good to have a test where the tags were an actual array of strings rather than an encoded JSON string.  I expect most users will populate their tags this way instead of escaping double-quotes in their serviceconfig file.